main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;
+const frame = 60;

```

Comment:
What's this mean? 60 frames per second? If so, maybe this should be `fps` or something instead of `frame`